### PR TITLE
find the error

### PR DESCRIPTION
**Pull request by veer-Singh**
please check out the errors @ do-em-dev

### CODE REVIEW

**Comment by veer-Singh** [comment]
plese comment @do-em-dev

**Comment by do-em-dev** [comment]
Ok, I will @veer-Singh

File: main.c [hunk 1]
```diff
@@ -101,20 +101,20 @@ int main(void)
...
 
   /* -3- Toggle PG.6, PG.8, PI.9 and PC.7 IOs in an infinite loop */
   while (1)
   {
-    HAL_GPIO_TogglePin(GPIOG, GPIO_PIN_6);
+   // HAL_GPIO_TogglePin(GPIOG, GPIO_PIN_6);
     /* Insert delay 100 ms */
-    HAL_Delay(1000);
-    HAL_GPIO_TogglePin(GPIOG, GPIO_PIN_8);
+  //  HAL_Delay(1000);
```
Comment (do-em-dev) [inline main.c:116]: DONT DO THAT

**Review by do-em-dev — COMMENTED** [review]
pLEASE CHECK

File: main.c (hunk 1 above)
Comment (veer-Singh) [reply main.c:116]: ok seir

Comment (do-em-dev) [reply main.c:116]: Theek HAI

**Pull request closed without merging**